src/components/Modal/Modal.module.css

```
@@ -26,7 +26,7 @@
 
 .overlay {
   position: fixed;
-  top: 0;
+  top: 0!important;
```

It could be great to comment why you needed to add this rule
```suggestion
  top: 0!important; /* hack to avoid the rules replaced by broken rule inset in chrome < 69  */
```